hypothesis-python/src/hypothesis/internal/conjecture/data.py

```diff
@@ -647,6 +677,8 @@ def draw_bits(self, n, forced=None):
         buf = hbytes(buf)
         result = int_from_bytes(buf)
 
+        self.observer.draw_bits(n, forced is not None, result)
```

Comment:
Where does `self.observer` get a `draw_bits` method from?

Reply:
The base `DataObserver` has a `draw_bits` method that does nothing.

`TreeRecordingObserver` has a `draw_bits` method that records the draw and its result in the tree.